src/DiscoveryServerManager.cpp

```diff
@@ -35,7 +35,7 @@ using namespace eprosima::fastrtps::rtps;
 using namespace eprosima::fastdds::rtps;
 using namespace eprosima::discovery_server;
 
-// non exported from fast-RTPS (watch out they may be updated)
+// non exported from Fast-DDS (watch out they may be updated)
 namespace eprosima {
 namespace fastrtps {
```

Comment:
Can you include in the checklist to rename this namespace at some point in this repository? Thanks!

Reply:
sure!